src/main/java/com/minecolonies/coremod/entity/ai/mobs/AbstractArcherBarbarian.java

```diff
@@ -0,0 +1,308 @@
+package com.minecolonies.coremod.entity.ai.mobs;
+
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.entity.EntityCitizen;
+import net.minecraft.entity.*;
+import net.minecraft.entity.ai.*;
+import net.minecraft.entity.monster.EntityMob;
+import net.minecraft.entity.passive.EntityWolf;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.entity.passive.EntityWolf'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)